src/Component/Inputs/TextInput/TextInput.tsx

```diff
@@ -39,9 +39,13 @@ export const TextInput = (props: TextInputProps) => {
         type="text"
         name={name}
         value={value}
+        data-testid="input-text"
```

Comment:
```suggestion
        data-testid="TextInput__text-input"
```
It would be useful for us to use component names as prefixes. They will be useful for cypress tests as well